test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -240,107 +155,133 @@ func schedulePods(config *testConfig) int32 {
 	}
 }
 
-// mutateNodeSpec returns the strategy needed for creation of nodes.
-// TODO: It should take the nodespec and return the modified version of it. As of now, returning the strategies for backward compatibilty.
-func (na nodeAffinity) mutateNodeSpec(numNodes int) []testutils.CountToStrategy {
-	numGroups := na.numGroups
-	nodeAffinityKey := na.nodeAffinityKey
-	nodeStrategies := make([]testutils.CountToStrategy, 0, numGroups)
-	for i := 0; i < numGroups; i++ {
-		nodeStrategies = append(nodeStrategies, testutils.CountToStrategy{
-			Count:    numNodes / numGroups,
-			Strategy: testutils.NewLabelNodePrepareStrategy(nodeAffinityKey, fmt.Sprintf("%v", i)),
-		})
+// getBaseNodeTemplate returns the base node template.
```

Comment:
TBH, I am kind of against this to be used as constant because this base template could be configurable in future. Like CPU, memory etc. I can add a TODO fo this.(like user can provide the inputs or if clusterloader drives the whole thing, it can invoke this function with CPU, memory etc). But I am open to other options too.